### PR TITLE
Feature/latest versions

### PR DESCRIPTION
**Pull request by salmma**
Please set the latest version tags of your components :)
Therefore, if not already done, make new releases of your components (containing the latest port adaptions).

### CODE REVIEW

File: db/Dockerfile [hunk 1]
```diff
@@ -8,13 +8,13 @@ ENV QC_ATLAS_CONTENT_REPOSITORY_BRANCH "master"
 ENV QC_ATLAS_SUBFOLDER_CONTENT_BACKUP_FILES "example-data/SQL/backup-files"
 ENV NISQ_CONTENT_REPOSITORY_URL "git@github.com:UST-QuAntiL/nisq-analyzer-content.git"
 ENV NISQ_CONTENT_REPOSITORY_PATH "/var/nisq/testdata"
-ENV NISQ_CONTENT_REPOSITORY_BRANCH "v1.1.0"
+ENV NISQ_CONTENT_REPOSITORY_BRANCH "master"
```
Comment (lharzenetter) [inline db/Dockerfile]: Isn't there also a new version available?

File: docker-compose.yml [hunk 2]
```diff
@@ -21,63 +21,63 @@ services:
       # use this volume to save the postgres data
       # - ./postgres-data:/var/lib/postgresql/data
     ports:
-      - '5432:5432'
+      - '5060:5060'
     networks:
       - default
   qc-atlas-ui:
-    image: planqk/qc-atlas-ui:v2.0.0
+    image: planqk/qc-atlas-ui:latest
```
Comment (lharzenetter) [inline docker-compose.yml]: Release missing

File: docker-compose.yml [hunk 3]
```diff
@@ -21,63 +21,63 @@ services:
...
+      QPROV_PORT: 5020
     ports:
       - '80:80'
     volumes:
       - ./config:/opt/init-config.d
     networks:
       - default
   qc-atlas:
-    image: planqk/atlas:v2.0.0
+    image: planqk/atlas:latest
```
Comment (lharzenetter) [inline docker-compose.yml]: Release missing

File: docker-compose.yml [hunk 4]
```diff
@@ -21,63 +21,63 @@ services:
...
       POSTGRES_PASSWORD: planqk
       POSTGRES_DB: planqk
     ports:
-      - "8080:8080"
+      - "6626:6626"
     networks:
       - default
   pattern-atlas-api:
-    image: patternpedia/patternrepo-api:v1.0.5
+    image: patternpedia/patternrepo-api:latest
```
Comment (lharzenetter) [inline docker-compose.yml]: Release missing

File: docker-compose.yml [hunk 5]
```diff
@@ -87,64 +87,64 @@ services:
...
+      PYTKET_PORT: 5015
     ports:
-      - '8081:8080'
+      - '5010:5010'
     networks:
       - default
   # used by qc-atlas-ui to render latex
   latex-renderer:
-    image: planqk/latex-renderer:v1.0.1
+    image: planqk/latex-renderer:latest
```
Comment (lharzenetter) [inline docker-compose.yml]: Release missing

**Review by lharzenetter — CHANGES_REQUESTED** [review]

File: db/Dockerfile (hunk 1 above)
Comment (salmma) [reply db/Dockerfile]: No, there is no newer version. But I will set ist to the latest v1.1.0 version.

Comment (lharzenetter) [reply db/Dockerfile]: In this case, revert this change please.

File: docker-compose.yml [hunk 6]
```diff
@@ -10,74 +10,74 @@ services:
...
+      QPROV_PORT: 5020
     ports:
       - '80:80'
     volumes:
       - ./config:/opt/init-config.d
     networks:
       - default
   qc-atlas:
-    image: planqk/atlas:v2.0.0
+    image: planqk/atlas:v3.1.0
```
Comment (salmma) [inline docker-compose.yml]: @manuwei don't you mean qc-atlas-ui? **- done** :)

**Review by salmma — COMMENTED** [review]
please adapt the version at qc-atlas-ui

File: docker-compose.yml (hunk 6 above)
Comment (salmma) [reply docker-compose.yml]: @manuwei is it also possible to rename patternpedia in the db folder, e.g. in the setup-patternatlas.sh to completely remove the name?

File: docker-compose.yml [hunk 7]
```diff
@@ -10,74 +10,74 @@ services:
...
       # specifies user:pw for each container
-      POSTGRES_USERS: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia|qprov:qprov'
+      POSTGRES_USERS: 'planqk:planqk|nisq:nisq|patternatlas:patternatlas|qprov:qprov'
       # specifies db name for each container
-      POSTGRES_DATABASES: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia|qprov:qprov'
+      POSTGRES_DATABASES: 'planqk:planqk|nisq:nisq|patternatlas:patternatlas|qprov:qprov'
       ATLAS_DB: planqk
       NISQ_DB: nisq
-      PATTERNPEDIA_DB: patternpedia
+      PATTERNPEDIA_DB: patternatlas
```
Comment (lharzenetter) [inline docker-compose.yml]: Shouldn't we also rename the db name?

File: docker-compose.yml (hunk 6 above)
Comment (manuwei) [reply docker-compose.yml]: Yes, we should do that

**Comment by manuwei** [comment]
changes related to the latex-renderer are still missing, I'll look into that today

File: docker-compose.yml (hunk 7 above)
Comment (manuwei) [reply docker-compose.yml]: you're right, we should do that as well

**Comment by manuwei** [comment]
I added the requested changes --> do to renaming the enviroment variable PATTERNPEDIA_DB--> PATTERNATLAS_DB, make sure to rebuild the db (docker-compose build db) for testing or using the setup for the first time :)

**Review by lharzenetter — APPROVED** [review]
LGMT :)

**Pull request merged**